src/wordpress-playground/index.tsx

```diff
@@ -297,3 +299,104 @@ function zipNameToHumanName(zipName) {
 }
 
 main();
+
+async function overwriteFile() {
+	const targetFile = document.getElementById('target-file').value;
+	const targetContent = document.getElementById('target-content').value;
+	console.log(targetFile);
+	console.log(targetContent);
+	await workerThread.writeFile(targetFile, targetContent);
+}
+
+async function generateZip() {
+	await workerThread.run({
```

Comment:
Let's inspect the output and throw an error with the stderr info if the code errors out for any reason (if `(await workerThread.run()).exitCode` is not 0)

Reply:
I added this error handling for each custom call to `workerThread.run()`. How does that look?

Reply:
Thank you, it looks great. Now I'm starting to wonder whether the `run()` method should just throw in case of errors, but that's out of scope for this PR. Good work here!